### ISSUE TITLE
Excel: NVDA fails to list comments in Elements List if a comment is on merged cells

### ISSUE THREAD

**Issue by michaelDCurran** [nvaccess/nvda#5704]
When NVDA searches Excel for cells containing comments, it finds all cells, no matter if they are merged or not. Only the first cell in the merged cells contains a valid comment, the rest have a comment property of None. We must ignore these.
